ArticleTemplates/assets/js/modules/messenger.js

```diff
@@ -0,0 +1,198 @@
+/* global Promise */
```

Comment:
@regiskuckaertz unfortunately we have to support Android 4.1 onwards and Promises aren't supported by Android until 4.4.4 http://caniuse.com/#feat=promises :confounded: